vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt

```diff
@@ -236,11 +236,9 @@ class WidgetFragment @Inject constructor() :
                 val context = requireContext()
                 val intent = Intent.parseUri(url, Intent.URI_INTENT_SCHEME)
                 if (intent != null) {
-                    val packageManager: PackageManager = context.packageManager
-                    val info = packageManager.resolveActivity(intent, PackageManager.MATCH_DEFAULT_ONLY)
```

Comment:
@ganfra can you tell me why we had this code please?

Reply:
I guess it was cleaner than doing try/catch? 
For the issue see: https://stackoverflow.com/questions/62535856/intent-resolveactivity-returns-null-in-api-30

Reply:
yes, it's cleaner than try catch, but I was wondering about the usage of `PackageManager.MATCH_DEFAULT_ONLY`